product_template_multi_link/views/product_template_link_view.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id='product_template_link__search' model='ir.ui.view'>
+        <field name="name">Product.template.link.search</field>
+        <field name="model">product.template.link</field>
+        <field name="arch" type="xml">
+            <search string="Product template links">
+                <field
+                    name="left_product_tmpl_id"
+                    string="Product"
+                    filter_domain="['|',('right_product_tmpl_id','ilike',self),('left_product_tmpl_id','ilike',self)]"
+                />
+                <field name="type_id" />
+                <group expand="1" string="Group By">
+                    <filter
+                        name="groupby_type"
+                        context="{'group_by' : 'type_id'}"
+                        string="Type"
+                    />
+                </group>
+            </search>
+        </field>
+    </record>
+    <!-- Technical Note: domain=[('id', '!=', 0)] is set on both template fields
+    to enable research by name for product templates. Otherwise, the search will
+    be done on product.product table, that is not the attempted behaviour
+    See odoo/addons/product/models/product_template.py // def name_search()
+    -->
+    <record id="product_template_link_tree" model="ir.ui.view">
+        <field name="name">Product.template.link.list</field>
+        <field name="model">product.template.link</field>
+        <field name="arch" type="xml">
+            <list editable="top">
+                <field
+                    name="left_product_tmpl_id"
+                    context="{'name_search_default_code': True}"
+                />
+                <field name="type_id" />
+                <field
+                    name="right_product_tmpl_id"
+                    context="{'name_search_default_code': True}"
+                />
+            </list>
+        </field>
+    </record>
+    <record id="product_template_link_kanban" model="ir.ui.view">
+        <field name="name">Product.template.link.kanban</field>
+        <field name="model">product.template.link</field>
+        <field name="arch" type="xml">
+            <kanban>
```

Comment:
Kanban view is broken.
![image](https://github.com/user-attachments/assets/ae6b4885-c407-4ef1-b083-78196d50b15f)